Add Accedian MetroNID OS Support

thecityofguanyu opened the pull request:
Committer: <thecityofguanyu@outlook.com>

 On branch thecityofguanyu/add-accedian-os
 Changes to be committed:
	new file:   html/images/os/accedian.png
	modified:   includes/definitions.inc.php
	new file:   includes/discovery/os/metronid.inc.php
	new file:   includes/discovery/processors/metronid.inc.php
	new file:   includes/discovery/sensors/temperatures/metronid.inc.php
	new file:   includes/polling/os/accedian.inc.php
	new file:   mibs/ACCEDIAN-SMI.mib
	new file:   mibs/ACD-ALARM-MIB.mib
	new file:   mibs/ACD-CFM-MIB.mib
	new file:   mibs/ACD-DESC-MIB.mib
	new file:   mibs/ACD-DISCOVERY-MIB.mib
	new file:   mibs/ACD-FILTER-MIB.mib
	new file:   mibs/ACD-PAA-MIB.mib
	new file:   mibs/ACD-POLICY-MIB.mib
	new file:   mibs/ACD-PORT-MIB.mib
	new file:   mibs/ACD-REGULATOR-MIB.mib
	new file:   mibs/ACD-SA-MIB.mib
	new file:   mibs/ACD-SFP-MIB.mib
	new file:   mibs/ACD-SHAPER-MIB.mib
	new file:   mibs/ACD-SMAP-MIB.mib
	new file:   mibs/ACD-TID-MIB.mib

laf commented on includes/discovery/os/metronid.inc.php:
```diff
@@ -0,0 +1,20 @@
+
+if (!$os) {
+    if (preg_match('/AMN-/', $sysDescr)) {
+        $os = 'accedian';
```
Is this the right name for the OS? Accedian is the company name and detection should be named after what the OS is.

laf commented on includes/discovery/processors/metronid.inc.php:
```diff
@@ -0,0 +1,31 @@
+
+
+if ($device['os'] == 'accedian') {
+ if ($device['version'] == 'AEN_5.3.1_22558') { 
```
Indents are four spaces.

laf commented:
Slight formatting issue in includes/discovery/processors/metronid.inc.php

Also query about the $os name.

thecityofguanyu commented:
@laf 

>Slight formatting issue in includes/discovery/processors/metronid.inc.php

Fixed in latest commit.

>Also query about the $os name.

Accedian made it a bit unclear here. The closest I can find to an OS name is 'AEN', which shows under firmware version. I've update it to reflect that instead of Accedian, also fixed in latest commit.

laf commented:
Didn't spot this the first time round but the files metronid.inc.php need updating to be the $os.inc.php

thecityofguanyu commented:
@laf 

>Didn't spot this the first time round but the files metronid.inc.php need updating to be the $os.inc.php

Fixed.
